tests/test_rbdigital.py

```diff
@@ -241,6 +241,21 @@ def queue_initial_patron_id_lookup(self):
         )
         self.api.queue_response(status_code=200, content=patron_datastr)
 
+    def queue_fetch_patron_bearer_token(self):
+        """Queue responses for the API calls used to obtain a patron
+        bearer token.
+
+        RBDigitalAPI.fetch_patron_bearer_token requires three API calls.
+        This method makes it easier and less error-prone to set up for that.
+        """
+        for filename in (
+            "response_patron_info_found.json",
+            "response_patron_internal_id_found.json",
+            "response_patron_bearer_token_success.json",
+        ):
+            datastr, datadict = self.api.get_data(filename)
+            self.api.queue_response(status_code=200, content=datastr)
+
```

Comment:
This is fine, but I want to make sure you know about a different solution to a similar problem in core/overdrive.py:MockOverdriveAPI: we override the method that asks for the token (`token_post`), eliminating the need to explicitly queue this stuff on most of the tests.

Reply:
Oh, that's handy. I'll keep that in mind.